apps/marketplace/components/BriefChoice/BriefChoice.js

```diff
@@ -1,79 +1,85 @@
-      briefType: type
-    })
-  }
-
-  render() {
-    return (
-      <div className={styles.container}>
-        <div className="row">
-          <div className="col-xs-12">
-            <h1 className="au-display-xl">Start a New Brief</h1>
-            <p>Create a new opportunity for a:</p>
-          </div>
-        </div>
-        <div className="row">
-          <div className={`${styles.choice} col-xs-12`}>
-            <AUradio
-              id="choice-digital-specialist"
-              label="Digital Specialist"
-              name="brief_choice"
-              value="digital-professionals"
-              onChange={() => {
-                this.handleChange('digital-professionals')
-              }}
-            />
-            <p>Get seller responses to your criteria plus candidate resumes for shortlisting.</p>
-          </div>
-        </div>
-        <div className="row">
-          <div className={`${styles.choice} col-xs-12`}>
-            <AUradio
-              id="choice-digital-outcome"
-              label="Digital Outcome"
-              name="brief_choice"
-              value="digital-outcome"
-              onChange={() => {
-                this.handleChange('digital-outcome')
-              }}
-            />
-            <p>
-              Get a summary of seller responses to your criteria, in spreadsheet format, for shortlisting. Once your
-              shortlist is complete you can request sellers to provide full proposals for you to evaluate.
-            </p>
-          </div>
-        </div>
-        <div className="row">
-          <div className="col-xs-12">
-            <a href={`/buyers/frameworks/digital-marketplace/requirements/${this.state.briefType}`}>
-              <button className="au-btn" disabled={!this.state.briefType && 'disabled'}>
-                Continue
-              </button>
-            </a>
-          </div>
-        </div>
-        <div className="row">
-          <div className="col-xs-12">
-            <a href={`${rootPath}/buyer-dashboard`}>Return to dashboard</a>
-          </div>
-        </div>
+const BriefChoice = () => (
+  <div className={styles.container}>
+    <div className={`row ${styles.headingRow}`}>
+      <div className="col-xs-12">
+        <AUheading level="1" size="xl">
+          Create a new brief
+        </AUheading>
+      </div>
+    </div>
+    <div className={`row ${styles.flex}`}>
+      <div className={`col-xs-12 col-md-3 ${styles.flexColumn}`}>
+        <span />
+        <AUheading level="2" size="md">
+          Digital outcome
+        </AUheading>
+        <p>Find a team to help you work towards a digital project or goal, for example, a website.</p>
+        <ul className={styles.tickList}>
+          <li>Receive written proposals from sellers.</li>
+          <li>Request case studies, work history, references and/or presentations.</li>
+        </ul>
+        <p className={styles.flexGrow}>
+          <strong>Receive responses within:</strong>
```

Comment:
Does this need to be in a `<strong>` tag?  Maybe just style with `font-weight`?

Reply:
Because the `1 - 2 weeks` part isn't bold, this would require adding some tag around the text anyway e.g. `span` to add the class for the font-weight. Using `strong` also means it will use default uikit bold text styling.